examples/2D_zero_circ_vortex/case.py

```diff
@@ -0,0 +1,122 @@
+import math
+import json
+
+# Numerical setup
+Nx = 250
+Ny = 125
+dx = 40./(1.*(Nx+1))
+
+M_inf = 0.3
+Mv = 0.1
+
+p_inf = 101325
+rho_inf = 1
+gam = 1.4
+
+c = math.sqrt(gam*(p_inf) / rho_inf)
+u_inf = M_inf * c
+cfl = 0.3
+mydt = cfl * dx / c 
+Tfinal = 150*1 / u_inf
+Nt = int(Tfinal/mydt)
+
+# Configuring case dictionary
+print(json.dumps({
+    # Logistics ================================================================
```

Comment:
It might be worthwhile to remove the "distracting characters," as mentioned in #609, to save work for whoever fixes it down the road and avoid creating more problems.

Reply:
I think these are fine since it will likely be a single find/replace that removes all of these distracting characters from all the case files (and elsewhere.